### DISCUSSION TITLE
Request to build a VISX example for D3 Hierarchical Bar Chart.

### DISCUSSION THREAD

**Discussion by mihir478** [airbnb/visx#1930] [General]
[Hierarchical Bar Chart](https://observablehq.com/@d3/hierarchical-bar-chart)

We are building data visualizations with VISX and are requesting a basic example for D3's hierarchical bar chart. I haven't found anything similar while searching the internet. If we have a basic example, I can customioze it for our use.

**Comment by cdx111** [comment]
I have a similar at here https://stackblitz.com/edit/react-1qtnwd?file=src%2FApp.js
